tasks_management/gql_queries.py

```diff
@@ -69,11 +69,14 @@ class Meta:
         connection_class = ExtendedConnection
 
     def resolve_current_entity_data(self, info):
-        entity = BenefitPlan.objects.first()
+        entity = self.entity
         if entity:
             serialized_json = _convert_to_serializable_json(entity)
             return serialized_json
-        return {}
+        return "{}"
```

Comment:
Please create constant either outside of classes or in that method. Would be more readable.

Reply:
done